index.js

```diff
@@ -216,14 +216,18 @@ Peer.prototype.signal = function (data) {
 
 Peer.prototype._addIceCandidate = function (candidate) {
   var self = this
-  try {
-    self._pc.addIceCandidate(
-      new self._wrtc.RTCIceCandidate(candidate),
-      noop,
-      function (err) { self._destroy(err) }
-    )
-  } catch (err) {
-    self._destroy(new Error('error adding candidate: ' + err.message))
+  if (!self._pc || !self._pc.remoteDescription.type) {
+    // Add candidate to queue
```

Comment:
Can you place a breakpoint here to see if this is ever reached? That would suggest the conditions are not identical. I would, but I can't seem to reproduce ICE failures at the moment.